Tests/src/test/scala/equellatests/pages/LoginNoticePage.scala

```diff
@@ -0,0 +1,74 @@
+package equellatests.pages
+
+import com.tle.webtests.framework.PageContext
+import equellatests.browserpage.NewTitledPage
+import org.openqa.selenium.{Keys, WebElement}
+
+case class LoginNoticePage(ctx: PageContext) extends NewTitledPage("Login Notice Editor", "page/loginconfiguration") {
+
+  def preNoticeApplyButton:WebElement = findElementById("preApplyButton")
```

Comment:
Typically we don't want to expose the WebElement's directly so probably best set all these to "private"